src/Cake.Common/Tools/DupFinder/DupFinderSettings.cs

```diff
@@ -91,5 +91,15 @@ public sealed class DupFinderSettings : ToolSettings
         /// Gets or sets a value indicating whether to show duplactes text in the report.
         /// </summary>
         public bool ShowText { get; set; }
+
+        /// <summary>
+        /// Gets or sets an optional flag for if should throw exception on finding duplicates
+        /// </summary>
+        public bool? ThrowOnDuplicates { get; set; }
+
+        /// <summary>
+        /// Gets or sets a value indicating whether analysis of results file should be completed
+        /// </summary>
+        public bool PerformResultsFileAnalysis { get; set; }
```

Comment:
The name and description of this property doesn't explain much and was a bit unclear to me. After looking at the usage a better description would be `AnalyzeOutputResult` and the description could be: "Gets or sets a value indicating whether the file-result should be analyzed."